launch_checklist.module

```diff
@@ -40,12 +38,31 @@ function launch_checklist_checklistapi_checklist_items() {
   if ($moduleHandler->moduleExists('webform')) {
```

Comment:
For module we don't know are installed, we want to define a variable first that is the project page for the module.

Then, using the module_handler service, we can figure out if the module is enabled, then set the route inside the Drupal install if it is.

Is this the best way to do this?